App.js

```diff
@@ -4,8 +4,29 @@ import { Asset } from 'expo-asset';
 import * as Font from 'expo-font';
 import React, { useState } from 'react';
 import { Platform, StatusBar, StyleSheet, View } from 'react-native';
+import * as Sentry from '@sentry/browser';
```

Comment:
I would suggest using `sentry-expo` instead. There might be some compatibility issues with `@sentry/browser`. 

Ref: https://docs.expo.io/versions/latest/guides/using-sentry/

Reply:
@JustinMi I agree with @averyyip -- let's use sentry-expo, it's essentially a wrapper around the `@sentry/react-native` with added functionality to make it easier to integrate with local development for expo